README.md

```diff
@@ -30,6 +30,129 @@ services.config(({app, services}) => {
 services.start();
 ```
 
+### Using as CLI with lb Architecture
+`@labshare/services` is a package for loading APIs. These APIs can be single on multiple mounted over a single thread.
+This package supports 2 modes:
+- services: LabShare API Services Architecture.
+- lb: LabShare API LoopBack Support Architecture.
+
+This can be defined at the .labsharerc file ( if it is not present, it will use services mode).
+
+```sh
+{
+  "mode": "lb", // or services
+  "apis":[{"name":"api"}]
+}
+
+```
+
+#### Services
+LabShare API Services Architecture can load both API modes, you will need to add the config.json configuration for services APIs and the config folder for lb APIs.
+The settings for starting the server will be taken from  the config.js configuration section `services.listen`
+The APIs that need to be loaded should be defined as package dependencies at the package.json
+  "packageDependencies": [
+    "@labshare/services",
+    "@labshare/facility"
+  ],
+Now you can start the app as lsc services start.
+#### lb
+LabShare API LoopBack Support Architecture will only load lb API mode.
+In order to use it you will need to have the following configuration:
+npm i @labshare/services  at the host project
+add a .labsharerc file with the following configuration
+```sh
+{
+  "mode": "lb",
+  "apis":[{"name":"api" , packare:"@labshare/facility-api" , configAlias:"facility"}]
+}
+
+```
+ 
+Where:
+- mode: it sets the mode to lb
+- apis: an array of APIs that will be loaded thru @labshare/services
+Each of the APIs will have the following configuration:
+
+- name: API name - it will be the use for retrieving the API configuration.
+- packare: npm package that will contain the API, if it is not defined it will load the local's project API.
+- configAlias: if defined it will load the API configuration with the given value.
+
+The lb API project should have:
+a config folder, this will need to have the API configuration, for example:
+```sh
+API - logging API
+name: logging
+package: @labshare/logging-api
+```
+```sh
+module.exports = {
```

Comment:
Unclear what this object refers to. It should be clarified that it is an example node-config config file under the config folder.